<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerFrame</class>
 <widget class="QFrame" name="LayerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>52</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>52</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <property name="lineWidth">
   <number>2</number>
  </property>
  <property name="midLineWidth">
   <number>1</number>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>5</y>
     <width>191</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ui_layerFrameLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="ui_isVisible">
      <property name="text">
       <string>isVisible</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="ui_isLock">
      <property name="text">
       <string>isLock</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="ui_nameInput"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ui_nameLabel">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
